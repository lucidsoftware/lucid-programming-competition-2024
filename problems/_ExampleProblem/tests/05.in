version https://git-lfs.github.com/spec/v1
oid sha256:28bad5c3db3124157b407468e0dc39f4a43573c79e5d283161bf2d931562ca79
size 404
