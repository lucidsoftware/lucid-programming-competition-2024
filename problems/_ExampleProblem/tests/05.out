version https://git-lfs.github.com/spec/v1
oid sha256:6bdf6e37dc327165fa4db779b911b98b6eba38852a9cd4a97cdd6be810a88357
size 8
